field.go

```diff
@@ -108,9 +107,11 @@ func Time(key string, val time.Time) Field {
 	return Float64(key, timeToSeconds(val))
 }
 
-// Error constructs a Field that stores err.Error() under the key "error". This is
-// just a convenient shortcut for a common pattern - apart from saving a few
-// keystrokes, it's no different from using zap.String.
+// Error constructs a Field that stores err.Error() under the key "error". If
+// passed a nil error, it returns a no-op field.
+//
+// This is just a convenient shortcut for a common pattern - apart from saving a
+// few keystrokes, it's no different from a nil check and zap.String.
```

Comment:
sure, in that case we probably don't want to say it's the same as `zap.String`, maybe that it's similar but this is lazy.